### PR TITLE
Add DIYgod

### PR DESCRIPTION
**Pull request by DIYgod**
Thanks! :)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Amas is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym).
 - [Blake Winton](https://github.com/bwinton/ama) - UX Engineer for Mozilla Firefox, and the author of Whimsy.
 - [Darrin Henein](https://github.com/darrinhenein/ama) - Design Lead for Firefox Mobile at Mozilla, creator of Lastronaut.
 - [Cassie McDaniel](https://github.com/cassiemc/ama) - Design Director for Mozilla Foundation, cofounder of Women&&Tech, speaker and writer.
+- [DIYgod](https://github.com/DIYgod/ama) - Front-end Engineer for Linux Deepin, blogger of anotherhome.net.
```
Comment (arthurvr) [inline readme.md]: > DIYgod

Use your full name instead of your user name.

Comment (DIYgod) [inline readme.md]: Sorry about that, it has already been fixed.

**Comment by arthurvr** [comment]
Thanks. Merged!

**Comment by sindresorhus** [comment]
Per the [instructions](https://github.com/sindresorhus/amas/blob/master/create-ama.md) you need to enable the issue tracker.

**Comment by sindresorhus** [comment]
Hey. Want to ask you a favor. Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=
https://github.com/sindresorhus/ama/issues?q=

**Pull request closed without merging**